shared/src/hover/HoverOverlay.scss

```
@@ -14,13 +14,14 @@
 
     &__close-button {
         position: absolute;
-        top: 0;
-        right: 0;
+        top: 1px;
+        right: 1px;
         padding: 0.25rem;
-        border-radius: 0;
+        border-radius: 1rem;
```

This would mean the button is a circle, right? Was that your intention? If yes, I'd make it
```suggestion
        border-radius: 100%;
```

---

Updated in both places